flytekit/interactive/vscode_lib/decorator.py

```diff
@@ -459,8 +443,14 @@ def execute(self, *args, **kwargs):
         )
         child_process.start()
 
-        # 6. Register the signal handler for task resumption. This should be after creating the subprocess so that the subprocess won't inherit the signal handler.
-        signal.signal(signal.SIGTERM, resume_task_handler)
+        # 3. Prepare the interactive debugging Python script and launch.json.
+        prepare_interactive_python(self.task_function)  # type: ignore
+
+        # 4. Prepare the task resumption Python script
+        prepare_resume_task_python(child_process.pid)
```

Comment:
@pingsutw For my knowledge, is passing the `child_process.pid` the major part of this fix?

Reply:
yup, otherwise, the rust worker in the actor task will be killed when resuming the task